frame/evm/src/lib.rs

```diff
@@ -517,12 +518,14 @@ pub mod pallet {
 }
 
 /// Type alias for currency balance.
-pub type BalanceOf<T> =
-	<<T as Config>::Currency as Currency<<<T as Config>::AccountProvider as AccountProvider>::AccountId>>::Balance;
+pub type BalanceOf<T> = <<T as Config>::Currency as Currency<
+	<<T as Config>::AccountProvider as AccountProvider>::AccountId,
+>>::Balance;
```

Comment:
Why this change? A lot of diffs for seemingly no reason

Reply:
Fixed